src/yb/rpc/yb_rpc.cc

```diff
@@ -320,7 +329,12 @@ bool YBInboundCall::DumpPB(const DumpRunningRpcsRequestPB& req,
     resp->set_trace_buffer(my_trace->DumpToString(true));
   }
   if (req.get_wait_state()) {
-    resp->set_wait_state(wait_state()->ToString());
+    auto wait_state = this->wait_state();
+    if (wait_state) {
+      resp->set_wait_state(wait_state->ToString());
```

Comment:
Should the wait state be serialized to a PB instead of string? That way it would be easier in pggate side to parse it

Reply:
I have it in the next part. 

updated here.